nyc/events.py

```diff
@@ -3,12 +3,12 @@
 
 import pytz
 import requests
-from legistar.events import LegistarAPIEventScraper
+from legistar.events import LegistarAPIEventScraperZip
 from pupa.scrape import Event, Scraper
 
 from .secrets import TOKEN
 
-class NYCEventsScraper(LegistarAPIEventScraper, Scraper):
+class NYCEventsScraper(LegistarAPIEventScraperZip, Scraper):
```

Comment:
Need to put this down for today, but want to log: When I run `pupa update nyc events`, all of the API events are logging an `API event could not be found in web interface` message. This happens with a local `python-legistar` install from both of our branches. :-(

Reply:
Walking this back! I did something wrong.

Reply:
Walking back the walk back! This was a flaw in `python-legistar`, submitted a PR with a patch: https://github.com/opencivicdata/python-legistar-scraper/pull/98